arcgis-ios-sdk-samples/Scenes/Add a point scene layer/AddPointSceneLayerViewController.swift

```diff
@@ -0,0 +1,58 @@
+//
+// Copyright © 2019 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import UIKit
+import ArcGIS
+
+class AddPointSceneLayerViewController: UIViewController {
+
```

Comment:
This blank line violates the [Vertical Whitespace after Opening Braces](
https://github.com/realm/SwiftLint/blob/master/Rules.md#vertical-whitespace-after-opening-braces) linting rule. Please remove it.

Just out of curiosity, did you install SwiftLint? If so, what did you think of the experience? Did you find it helpful?

Reply:
I did installed SwiftLint and it is pretty useful in following good coding practices. But weirdly it stopped working till I restart my machine and clear cache.

Reply:
Good to know. I've never had that happen, but I'll keep an eye out for it.